AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Rekognition.swift

```diff
@@ -375,27 +379,31 @@ extension AWSPredictionsService: AWSRekognitionServiceBehavior {
                 dispatchGroup.leave()
                 return nil
             }
+
             guard let result = task.result else {
                 onEvent(.failed(.unknown(AWSRekognitionErrorMessage.noResultFound.errorDescription,
                                          AWSRekognitionErrorMessage.noResultFound.recoverySuggestion)))
                 errorOcurred = true
                 dispatchGroup.leave()
                 return nil
             }
+
             guard let moderationRekognitionLabels = result.moderationLabels else {
                 onEvent(.failed(.network(AWSRekognitionErrorMessage.noResultFound.errorDescription,
                                          AWSRekognitionErrorMessage.noResultFound.recoverySuggestion)))
                 errorOcurred = true
                 dispatchGroup.leave()
                 return nil
             }
+
             unsafeContent = !moderationRekognitionLabels.isEmpty
             dispatchGroup.leave()
             return nil
         }
         dispatchGroup.wait()
+
         if !errorOcurred {
```

Comment:
same note about `predictionsError`

Reply:
As discussed, I will refactor it in another PR